### PR TITLE
Add basic game model

### PR DESCRIPTION
**Pull request by nc-firthh**
This adds a very basic `Game` model and controller as a jumping off point for building our abstraction on top of the MLB api.

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -1,24 +1,46 @@
...
+This is a very simple rails app which serves JSON data pertaining to a baseball game.
 
-Things you may want to cover:
+## Getting Started
 
-* Ruby version
+### Prerequisites
 
-* System dependencies
+In order to run this application locally, you will need versions `2.5.1` of ruby, and `5.2.1` of rails installed on your machine, as well as the `bundler` gem.
```
Comment (pattertom) [inline server/README.md]: We should add a `.ruby-version` file that has `2.5.1` for people using rvm/rbenv

Comment (nc-firthh) [reply server/README.md]: There already is one?

Comment (pattertom) [reply server/README.md]: Oh I was looking for one in the root instead of `/server` 😂

**Comment by nc-firthh** [comment]
Gonna move this over to the baseball exercise.

**Pull request closed without merging**